DESCRIPTION

```diff
@@ -39,14 +39,14 @@ Copyright: This software is in the public domain because it contains materials
 Depends:
     R (>= 3.5.0)
```

Comment:
For what it's worth, httr2 (and the tidyverse) now require R >= 4.0, and in the spring will require R >= 4.1 . If you bump this to 4.1 (which, depending when you're planning on launching things, will be the minimum version to use the package anyway) then you can use the base pipe, which might make some of the query-building more fluid

Reply:
🥳I've been avoiding the 4.1 requirement, but if httr2 has it already....HELLO PIPES

Reply:
You could also re-export the magrittr pipe from httr2 (either directly or by adding magrittr as a direct dependency) if this is going out before the spring. Here's how httr2 does it:
https://github.com/r-lib/httr2/blob/main/R/utils-pipe.R